recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/resolver/FetchUsersFromExternalServices.java

```diff
@@ -52,6 +53,13 @@
 @ApplicationScoped
 public class FetchUsersFromExternalServices {
 
+    public static final String COUNTER_FAILURES = "notifications.recipients-resolver.failures";
+    public static final String COUNTER_SUCCESSES = "notifications.recipients-resolver.successes";
+    protected static final String COUNTER_TAG_USER_PROVIDER = "user-provider";
+    protected static final String COUNTER_TAG_USER_PROVIDER_RBAC = "rbac";
+    protected static final String COUNTER_TAG_USER_PROVIDER_MBOP = "mbop";
+    protected static final String COUNTER_TAG_USER_PROVIDER_IT = "it";
+
```

Comment:
I wonder if we should just have a single `notifications.recipients-resolver.requests` counter and a `successful` or `failed` tags, along with the already implemented `user-provider` tag :thinking: